app/controllers/protected_areas_controller.rb

```diff
@@ -90,10 +94,25 @@ def load_other_designations
     other_designations.reject { |pa| pa.id == @protected_area.id }
   end
 
-  TRANSBOUNDARY_SITES = "Transboundary sites".freeze
-  def load_networks
-    networks = @protected_area.networks.reject(&:designation)
-    # ensure that transboundary sites network always appears first
-    networks.sort { |a,b| a.name == TRANSBOUNDARY_SITES ? -1 : a.name <=> b.name }
+  # TODO: Methods, models, controllers, modules related to Networks are slated for removal
+
+  # TRANSBOUNDARY_SITES = "Transboundary sites".freeze
+  # def load_networks
+  #   networks = @protected_area.networks.reject(&:designation)
+  #   # ensure that transboundary sites network always appears first
+  #   networks.sort { |a,b| a.name == TRANSBOUNDARY_SITES ? -1 : a.name <=> b.name }
+  # end
+
+  def get_other_sites
+    return ProtectedArea.all_except(@protected_area).take(3) if @countries.length <= 1
+    ProtectedArea.all_except(@protected_area).transboundary_sites.take(3)
+  end
+
+  def determine_search_path(area)
+    if area.is_transboundary
+      search_areas_path(geo_type: 'transboundary')
```

Comment:
@stanleypliu  This is incorrect as the geo_type attribute is about sites VS country VS region, so it throws an error.
At most, this should either be the same as the url in the `else` statement, or should assign a filter similar to the greenlist one like: `filters[special_status][]=is_transboundary`.

@stacytalbot I remember that we decided to not include this filter in the search at some point; does this one need to come back now that we have the functionality?

Reply:
Admittedly I was unsure where it would go, my first thought was that it should be a special status, but then I thought that it's basically a variation of a site, so I classified it under `geo_type`

Reply:
@stacytalbot @Levia I've added transboundary as a special status now, instead of a `geo_type`